### PR TITLE
floodfill sol

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

OH MY GOD YOU HAVE NO IDEA HOW LONG THE JAVA SOL TOOK ME TO MAKE
LIKE IT WAS A BUG BECAUSE TWO `Integer`s CANT BE COMPARED WITH `==`
I'M ACTUALLY GONNA LOSE IT

image

### CODE REVIEW

File: solutions/gold/cf-1114D.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+The below code only runs in time if you submit with PyPy.
+
+</Warning>
+
+```py
+from itertools import groupby
+
+len_ = int(input())
+arr = [int(i) for i in input().split()]
+assert len(arr) == len_
```
Comment (ryanchou-dev) [inline solutions/gold/cf-1114D.mdx:137]:
```suggestion
```

(optional) just feels unnecessary to me 👀

**Review by ryanchou-dev — APPROVED** [review]
christmas is coming kevin :O

**Comment by SansPapyrus683** [comment]
god i love getting stressed to the point of literal physical pain every night

**Comment by TheGamingMousse** [comment]
> _Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._
> 
> * [x]  I have tested my code.
> * [x]  I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
> * [x]  I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
>   
>   * I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
>   * If changes are requested, I will re-request a review after addressing them.
> * [x]  I have linked this PR to any issues that it closes.
> 
> OH MY GOD YOU HAVE NO IDEA HOW LONG THE JAVA SOL TOOK ME TO MAKE LIKE IT WAS A BUG BECAUSE TWO `Integer`s CANT BE COMPARED WITH `==` I'M ACTUALLY GONNA LOSE IT
> 
> image

ap cs ahh mistake

**Review by TheGamingMousse — APPROVED** [review]

**Pull request merged**